run_script_templates/rnaseq_GRCh38_gencode_template

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+#SBATCH -A ngi2016001
+#SBATCH -p core
+#SBATCH -n 16
+#SBATCH -t 10-00:00:00
+#SBATCH -J rnaseq_PROJECT
+#SBATCH -e /proj/ngi2016001/nobackup/NGI/ANALYSIS/PROJECT/logs/rnaseq_%j.log
+#SBATCH -o /proj/ngi2016001/nobackup/NGI/ANALYSIS/PROJECT/logs/rnaseq_%j.log
+
+# Source NGI environment
+source /lupus/ngi/production/latest/conf/sourceme_upps.sh
+source activate NGI
+
+#set Nextflow env variables
+export NXF_WORK=/proj/ngi2016001/nobackup/NGI/ANALYSIS/PROJECT/work
+
+rnaseq -resume -c /lupus/ngi/production/latest/sw/upps_standalone_scripts/config/genomes_GRCh38_gencode.config --genome GENOME --reads '/proj/ngi2016001/nobackup/NGI/DATA/merged_fastqs_PROJECT/*_R{1,2}.fastq.gz' \
```

Comment:
Backslash before `--reads` so that it's easier to read maybe?

Reply:
Fixed